### DISCUSSION TITLE
Questions on Langevin Dynamics: Force Magnitude and Particle Mass Determination in HOOMD-blue

### DISCUSSION THREAD

**Discussion by idop11** [glotzerlab/hoomd-blue#1824] [Support] [answered]
Hello,

I have two questions regarding the use of Langevin dynamics in HOOMD-blue:

1. My system consists of Brownian particles with interactions, specifically the DPD conservative pair force. I want to determine the magnitude of this force (the parameter 'A') to be of the same order as the random force $F_R$ in the equation of motion for the Langevin equations. However, I'm a bit unsure how to determine the root mean square (RMS) of the magnitude of $F_R$, because it also includes a division by $\delta t$. Does it mean that the RMS of the random force over dt time (which I determine in my simulation) is $\sqrt{2dkT\gamma}$?
2. The second question concerns the mass in Langevin dynamics. Although it doesn't need to be determined, it affects the motion of the particles, and I want to understand how to determine its value. I understand that it should be related to the drag coefficient, $\gamma$, but I'm not sure how exactly. Could I get an explanation about this?

Thanks!

**Comment by joaander** [comment, marked as answer]
1. Understand that both Brownian and Langevin dynamics assume a decorrelated random force at *all* times. The $\delta t$ appears in the random force as a consequence of numerically approximating the solution with discrete time steps. You can read more about this in any good computational physics textbook.
2. Are you asking about Brownian or Langevin dynamics? You mention both, but they are very different. There is no mass in Brownian dynamics. For Langevin dynamics, you can set the mass to any value of your choosing (>0).
